src/site/content/en/blog/lighthouse-whats-new-6.0/index.md

```diff
@@ -0,0 +1,463 @@
+   additional improvements.
+-  **First CPU Idle has been deprecated** because it's not distinct enough from TTI. TBT and TTI
+   are the go-to metrics for interactivity now.
+-  CLS's weight is relatively low, though we expect to increase it in a future major version.
+
+**TODO(cjamcl)**: Make two images for v5/v6 scoring weights.
+
+### Shifts in scores {: #score-shifts }
+
+How do these changes affect the scores of real sites? We have published an
+[analysis](https://docs.google.com/spreadsheets/d/1BZFh7AyyaLHCj5LGAbrn3m72ysu4yv8okyHG-f3MoXI/edit?usp=sharing)
+of the score changes using two datasets: a [general set of
+sites](https://gist.github.com/connorjclark/8afe673d4e7c6e17204834a256e7caf1) and a
+[set of static sites](https://gist.github.com/connorjclark/0be52464887ae3a6f29ad5a798122e0c#file-readme-md)
+built with [Eleventy](https://www.11ty.dev/). In summary, ~20% of sites see noticeably higher
+scores, ~30% have hardly any change, and ~50% see a decrease of at least five points.
+
+The score changes can be broken down into three primary components:
+
+1. Score weight changes
+1. Bug fixes to underlying metric implementations
+1. Individual score curve changes
+
+Score weight changes and the introduction of three new metrics drove a majority of overall score
+changes. New metrics that developers have yet to optimize for carry significant weight in the v6
+performance score. While the average performance score of the test corpus in v5 was ~50, the average
+scores on the new Total Blocking Time and Largest Contentful Paint metrics were in the 30s. Together
+those two metrics account for 50% of the weight in the v6 performance score, so naturally a large
+percentage of sites saw decreases.
+
+Bug fixes to the underlying metric computation can result in different scores. This affects
+relatively few sites but can have sizable impacts in certain situations. Overall, ~8% of sites
+experienced a score improvement due to metric implementation changes and ~4% of sites saw a score
+decrease due to metric implementation changes. ~88% of sites were unaffected by these fixes. 
+
+Individual score curve changes also impacted the overall score shifts although very slightly. We
+periodically ensure that the score curve aligns with the observed metrics in the [HTTPArchive
+dataset](http://httparchive.org/). Excluding sites affected by major implementation changes, minor
+adjustments to the score curve for individual metrics improved the scores of ~3% of sites and
+decreased the scores of ~4% of sites. ~93% of sites were unaffected by this change.
+
+### Calculator {: #calculator }
+
+We've published a [scoring calculator](https://googlechrome.github.io/lighthouse/scorecalc/) to help
+you explore performance scoring. The calculator also gives you a comparison between version v5 and
+v6 scores. When you run an audit with Lighthouse 6.0, the report comes with a link to the calculator
+with your results populated.
+
+![image](lighthouse-scorecalc.png)
```

Comment:
this definitely needs a 1px border. it actually looks interactive with the screenshot right now.

Reply:
see note above about `w-screenshot`